gabarit/template_nlp/nlp_project/package_name/monitoring/mlflow_logger.py

```diff
@@ -70,6 +91,21 @@ def tracking_uri(self, uri:str) -> None:
         '''Set tracking uri'''
         mlflow.set_tracking_uri(uri)
 
+    @property
+    def experiment_id(self) -> str:
+        '''Experiment id. It can not be changed.'''
+        return self._experiment_id
+
+    @property
+    def experiment_name(self) -> str:
+        '''Experiment name. It can not be changed.'''
+        return self._experiment_name
+
+    @property
+    def artifact_uri(self) -> str:
+        '''Experiment artifact URI. It can not be changed.'''
+        return self._artifact_uri
+
```

Comment:
See above comment

Reply:
In combination with the @property decorator it helps to protect these attributes from being set.